### ISSUE TITLE
Elements with display:table-row not included in background color test

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#1588]
**Expected**: The following should pass the color-contrast rule

```html
<div style="background-color: #003; color: #eee; display: table-row;">
  <p>Good contrast</p>
</div>
```

**Actual**: The rule fails to pick up the background-color because it is on a `display: table-row` element. This is similar to how `<tr>` and `<thead>` elements aren't picked up by `elementsFromPoint`.

**Comment by dbjorge** [comment]
Note that this issue repros on Chromium (stable and dev) but not Firefox, as of writing. We are reasonably confident that the root cause is [Chromium bug 647594](https://bugs.chromium.org/p/chromium/issues/detail?id=647594). We already notified the Chromium team of the axe-core impact and they have already escalated the bug to P1 accordingly, tagged for release M-78.

**Comment by jeankaplansky** [comment]
Include as part of the overall improvements/fixes to color-contrast

**Comment by jeankaplansky** [comment]
https://docs.google.com/document/d/14PI0e3X9g6CBWtl86HReHrRxl-_zapZBE8HaKoOHrmE/edit#heading=h.cy2pxqhzo2cm

**Comment by padmavemulapati** [comment]
Throwing color-contrast voilation.
![image](https://user-images.githubusercontent.com/47409579/73331276-468fbf80-4289-11ea-8d80-8ae93345c5b3.png)

**Comment by atotic** [comment]
FYI: Chrome will not change elementFromPoint behavior for rows/sections. I've tested it in all modern browsers, and it seems that current Firefox matches Chrome's behavior.

Background color of a table cell is also determined by columns. This makes it impossible to determine true table background cell from javascript in general case. (unless there is an api I am not aware of). I suggest filling a csswg issue if this is a capability you need for accessiblity.
